app/api/data.py

```
@@ -0,0 +1,889 @@
+    # Get team info for keyword matching
+    team = await db.teams.find_one({"_id": team_id})
+    if not team:
+        return {"average_sentiment": 0.0, "total_mentions": 0}
+
+    # Get recent sentiment analyses (last 7 days)
+    week_ago = datetime.now() - timedelta(days=7)
+
+    # Search for sentiment analyses mentioning this team
+    team_keywords = [team["name"].lower(), team["abbreviation"].lower()] + team.get(
+        "aliases", []
+    )
```

Ruff (0.13.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)